app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <integer name="duration_anim_300_ms">300</integer>
```

Comment:
This integer doesn't appear to be used--please remove.

The gradient angle below also doesn't appear to be shared with anything. Let's only put values in shared locations if they're going to be used elsewhere (strings being the main exception since they need to be translated, and sometimes colors to help put context for the arbitrary value correspond to a color--this case doesn't require as much context since the place where the angle is used is pretty clear).

Reply:
Ditto above for the entire dimens.xml file, as well.

Reply:
its used in more than one place in activity transition animations

Reply:
Sorry, can you indicate which file? I couldn't find this used anywhere in the PR.

Reply:
In most of the files in anim folder i'am using it for example slide_right_in